config.yaml.dist

```diff
@@ -8,6 +8,9 @@ network:
   ip_range:
     start: 10.0.3.31
     end: 10.0.3.70
+  bridge_ip_range:
+    start: 12.0.3.31
```

Comment:
`12.0.3.0` is not a private ip range. let's use something like `192.168.10.1/24`. and there is no need to use `start`, `end` here. so let's use the full CIDR notation that is used in the networkd template.